.github/workflows/chromatic.yaml

```diff
@@ -3,67 +3,44 @@
 # Workflow name
 name: "chromatic"
 
-on:
-  # despite the name, this is triggered on both isuse comments *and* PR comments. We're only interested in PR comments for this Action.
-  issue_comment:
-    types: [created]
+# Event for the workflow
+on: push
 
 # List of jobs
 jobs:
   visual-testing:
     # Operating System
     runs-on: ubuntu-latest
-
+    env:
+      PACKAGE_REGISTRY_ACCESS_TOKEN: ${{ secrets.PACKAGE_REGISTRY_ACCESS_TOKEN }}
     # Job steps
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 0 # 👈 Required to retrieve git history
-
-      - name: Get most recent comment
-        if: ${{ github.event.issue.pull_request }} # despite the name "issue_comment", an "issue_comment" can either actually be an issue comment, or it might be a PR comment (even though issues and PRs are completely different things). This checks where this is a "PR comment" kind of "issue comment". You know like when sometimes a dog isn't a dog, but it's actually a cat so you need to check to make sure?
-        uses: peter-evans/find-comment@v1
-        id: latestComment
-        with:
-          issue-number: ${{ github.event.issue.pull_request }}
-          body-includes: ""
-          direction: last
-
-      - run: |
-          echo "runChromatic=${runChromatic}" >> $GITHUB_ENV
-        env:
-          runChromatic: ${{ contains(steps.latestComment.outputs.comment-body, 'run chromatic') }}
-
-      - run: echo "found run chromatic text in latest comment"
-        if: ${{ env.runChromatic == 'true' }}
-
       - uses: actions/cache@v2 # use of cache for node_modules saves usually > 1min of build time
-        if: ${{ env.runChromatic == 'true' }}
         with:
           path: "**/node_modules"
           key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-${{ hashFiles('**/package.json') }}
           restore-keys: |
             ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-
             ${{ runner.os }}-
-
       - uses: actions/setup-node@v1
-        if: ${{ env.runChromatic == 'true' }}
         with:
           node-version: "14.15.x"
-
       - name: Install package dependencies
-        if: ${{ env.runChromatic == 'true' }}
         run: |
-          yarn install --frozen-lockfile
-
-      # 👇 Adds Chromatic as a step in the workflow
+          yarn install --frozen-lockfile --ignore-scripts
+          # --ignore-scripts ignores the preinstall script that checks if the executor is logged into the github registry
```

Comment:
We will need to remove `--ignore-scripts` and implement the private registry authentication when we start using frontend-ops.

Reply:
Thanks for dropping in this comment. No doubt it will be a gotcha when we roll out FE-ops in this repo